src/opnsense/www/themes/opnsense/build/css/main.css

```diff
@@ -7561,3 +7561,20 @@ div[data-for*=help_for] {
   margin-top: 0px;
   margin-bottom: 0px;
 }
+
```

Comment:
In OPNsense, we have removed some CSS from `list-group-item` (which is part of Bootstrap). This causes them to appear incorrectly. Since I don't know why that CSS was removed, I wanted to play it safe and define `list-group-item-border` to allow for correct drawing.

Bootstrap doc: https://bootstrapdocs.com/v3.3.6/docs/components/#list-group

Reply:
what's the relation to this commit? (main.css is compiled, this will be overwritten if not in the sass file)

Reply:
I used `list-group-item`s in order to list the matching aliases, and I need them to have borders.